.travis.yml

```diff
@@ -1,5 +1,8 @@
 sudo: false
 language: node_js
+addons:
+    code_climate:
+        repo_token: $CODECLIMATE_REPO_TOKEN
```

Comment:
> where is this variable set?

https://travis-ci.org/openpgpjs/openpgpjs/settings

> codeclimate checks the code quality (with linters), but the code is instrumented and subject to code coverage in every repo, as we are calculating it in travis. then codeclimate will show the last one published.
> 
> what i suggest is then also to instrument the code and publish it only on the master branch; i will modify travis.sh accordingly if you are ok.

Sure. Do you want to merge this PR to master then? I will continue to work without coverage on the typed array branch and have set the node v4.2 and v5 travis jobs as required there.